services-manager/src/main/resources/application-debug.yaml

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 #
 
+eureka:
```

Comment:
I would have thought this should only be in the eureka profile and not in the debug profile. I assume we will want to use debug profile with both JVM and K8 deployments

Reply:
The logging profile can do two things:
* Change what level is used on startup
* Ping all available /loggers endpoints to change level at runtime

This second feature requires eureka.
In the future, there may be a k8s oir jvm 'mode' switch seperately that could also include helm install commands etc. but regardless, the debug profile switch should be expecting eureka to be running in jvm mode

Reply:
I'll pull it out, because there are some other changes required to the scripts